ql/src/test/queries/clientpositive/avro_proleptic_mixed_timestamp.q

```diff
@@ -7,13 +7,13 @@ stored as avro;
 
 INSERT INTO hybrid_table VALUES
 ('2012-02-21 07:08:09.123'),
-('2014-02-11 07:08:09.123'),
-('1947-02-11 07:08:09.123'),
+('2014-02-11 07:08:09.123456'),
+('1947-02-11 07:08:09.1234'),
 ('8200-02-11 07:08:09.123'),
-('1012-02-21 07:15:11.123'),
+('1012-02-21 07:15:11.12345'),
 ('1014-02-11 07:15:11.123'),
-('0947-02-11 07:15:11.123'),
-('0200-02-11 07:15:11.123');
+('0947-02-11 07:15:11.12345'),
+('0200-02-11 07:15:11.123456');
```

Comment:
Refrain from changing existing tests. Add a new test for timestamp-micros explicitly mentioning its column type.